rules/robot_players.tex

```diff
@@ -53,7 +53,13 @@ \subsection{Team Markers}
 \item Designs must be submitted to \url{rc-spl-tc@lists.robocup.org} for approval by \DTMdate{\JerseyApproveSubmissionDate}. If the team has jersey prototypes, they should submit close-up images of a robot wearing the jersey---these images should be taken from front, back, and side angles.  If the team has no prototypes, then designs depicting the expected jersey should be submitted.  If submissions show separated front and back halves of jerseys then the team must specify which halves are matched to form home and away jerseys.  All images and designs should be submitted in pdf or jpg format.
 \end{itemize}
 
-Each team must designate a ``home'' color and an ``away'' color for their \emph{field players} when asked about one month before RoboCup. Teams must wear their ``home'' jerseys when they are ``home'' (the first team listed on the schedule). Teams will wear their ``home'' jersey when they are ``away'' (the second team listed on the schedule) as well, unless either the head referee or the GameController program believes the jerseys of two competing teams are too similar.  In this case, the ``away'' team will then wear their ``away'' jersey.
+The jersey sets that are used in a game are decided by the head referee in accordance with the teams during the pre-game meeting (\cf~\cref{sec:referee_team_communication}), subject to the following conditions:
+\begin{itemize}
+  \item Both teams' \emph{field player} colors must be easily distinguishable from each other.
+  \item Both teams' \emph{goalkeeper} colors must be easily distinguishable from both teams' \emph{field player} colors.
+  \item The two \emph{goalkeeper} colors may only be similar when this is the case for all choices among the available jersey sets.
+\end{itemize}
+The head referee should respect the teams' wishes unless they violate the above conditions, in which case the head referee must bring about a solution.
```

Comment:
```suggestion
The head referee should respect the teams' wishes unless they violate the above conditions, in this case the head referee comes up with a solution.
```
Sounded a little bit weard for me. That is the only point here.

Reply:
I'm not 100% sure, but I think that "in which case" can join two clauses and "in this case" cannot. If you wanted "in this case", you'd have to split it into two sentences (or perhaps use a semicolon, but to me, a comma doesn't "feel" enough to separate the clauses). "in which case" is definitely a valid expression here.

For the second part, I agree that "bring about" is an uncommon phrase, as I didn't know it before I was looking for a translation of the German word "herbeiführen" (as in "der Schiedsrichter muss eine Lösung herbeiführen").